source/reference/options.txt

```diff
@@ -393,12 +393,25 @@ Authentication Options
    :option:`--password <--password>` option as the last option and leave out the
    argument.
 
+.. option:: --quiet
+
+   Skip all messages during startup (such as welcome messages and startup
+   warnings) and go directly to the prompt.
+
 .. option:: --username <username>, -u <username>
 
    Specifies a username with which to authenticate to a MongoDB database
    that uses authentication. Use in conjunction with the
    :option:`--password <--password>` and
-   :option:`--authenticationDatabase <--authenticationDatabase>` options.
+   :option:`--authenticationDatabase <--authenticationDatabase>`
+   options.
+
+.. option:: --verbose
+
+   Increases the verbosity of the output of the shell during the connection
```

Comment:
suggestion (non-blocking):

Consider "Increases the verbosity of the shell output..." May flow a bit better.